geometry_msgs/Pose pose
---
sensor_msgs/PointCloud raytrace
